coding/python_style_guide.rst

```diff
@@ -2,7 +2,7 @@
 DM Python Style Guide
 #####################
 
-This is the version 6.0 of the DM Python Coding Standard.
+` is the version 6.0 of the DM Python Coding Standard.
```

Comment:
What happened here?

Reply:
Oops. Good catch.

Reply:
Are we updating that number ever? I don't remember updating it when I've made previous changes.

Reply:
It's a hold-over from the Confluence page. Maybe A subject for another PR, but I'd say we should either drop it (the only standard that matters is always the current standard), or get in the matter of maintaining a change log if we actually do care about versioning the style guide.

Reply:
Good point. I'm not going to touch it now (the above was a mistake on my part), but we should probably remove it.